package.json

```diff
@@ -26,7 +26,8 @@
     "passport": "^0.4.0",
     "pg": "^7.12.1",
     "pg-hstore": "^2.3.3",
-    "sequelize": "^5.15.0"
+    "sequelize": "^5.15.0",
+    "uuid": "^3.3.3"
```

Comment:
Nice work @sa-ma , I think uuid now comes with sequelize, so we don't need an extra package

Reply:
I've removed it